.github/workflows/check-traceability.yml

```diff
@@ -0,0 +1,47 @@
+name: Check PR traceability
+
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+
+jobs:
+  check-pr-description:
+    name: Check PR description
+    runs-on: ubuntu-20.04
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+
+    - name: Fetch PR Description
+      env:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      run: echo "::set-output name=description::$(gh pr view ${{ github.event.pull_request.number }} | tr -d "\n\r" | tr -d ' ')"
+      id: pr
+
+    - name: Check PR validity
```

Comment:
Can we skip this if `gh pr view <Number> --json labels --jq 'any(.labels[].name; . == "let's do it!")'`

Reply:
I think every PR should be checked, possibly for us.